invenio_previewer/templates/semantic-ui/invenio_previewer/bottom.html

```diff
@@ -36,16 +33,6 @@ <h3 id="embedModalLabel">{{_('Embed code')}}</h3>
   body {
     height: 100%;
   }
-  #wrap {
```

Comment:
minor: Could all the CSS and JS be moved to css and js files instead of having them as snippets here? (I know it's not what the task is supposed to do, just in case it is easy :) )